Source/Server/Weapons/WPhoenix.cs

```diff
@@ -47,11 +47,8 @@ protected override void ShootOnce()
         // Determine projectile velocity
         Vector3D vel = Vector3D.FromActorAngle(client.AimAngle, client.AimAngleZ, PROJECTILE_VELOCITY);
 
-        // Move projectil somewhat forward
-        Vector3D pos = client.State.pos + Vector3D.FromActorAngle(client.AimAngle, client.AimAngleZ, PROJECTILE_OFFSET);
```

Comment:
Please note that `PROJECTILE_OFFSET` was different from `WPhoenix` (I mean it is equal to `4f` for everyone _except_ Phoenix) before this change.

Perhaps we should parametrize the `GetProjectileInitialPosition` method instead?